Source/Godunov.H

```diff
@@ -1155,6 +1128,11 @@ pc_artif_visc(
     }
   }
   flx(iv, UTEMP) = 0.0;
+#if NUM_AUX > 0
+  for (int n = 0; n < NUM_AUX; ++n) {
```

Comment:
really nitpicky, but all the other for loops are `n++` in this file. Can you change to that?

Reply:
All of them except for the one directly above this loop! I fixed both of them for you.